plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -138,9 +142,25 @@ protected void copyTableSchema(Connection connection, String catalogName, String
     protected String createTableSql(RemoteTableName remoteTableName, List<String> columns, ConnectorTableMetadata tableMetadata)
     {
         ImmutableList.Builder<String> tableOptions = ImmutableList.builder();
-        ClickHouseTableProperties.getEngine(tableMetadata.getProperties())
-                .ifPresent(value -> tableOptions.add("ENGINE =" + quoted(value)));
-        return format("CREATE TABLE %s (%s) %s", quoted(remoteTableName), join(", ", columns), join(", ", tableOptions.build()));
+        Map<String, Object> tableProperties = tableMetadata.getProperties();
+        ClickHouseEngineType engine = ClickHouseTableProperties.getEngine(tableProperties);
+        tableOptions.add("ENGINE = " + engine.getEngineType());
+        log.info("engine = " + engine);
```

Comment:
remember to either remove this (because the engine type by itself is meaningless in production logs) or add more information and log at debug.

Reply:
OK